src/Microsoft.Data/DataFrameTable.cs

```diff
@@ -138,6 +138,76 @@ public void RemoveColumn(string columnName)
             }
         }
 
+        private void ResizeByOneAndAppend(BaseColumn column, object value)
+        {
+            long length = column.Length;
+            column.Resize(length + 1);
+            column[length] = value;
+        }
+
+        public void Append(IEnumerable<object> row)
+        {
+            IEnumerator<BaseColumn> columnEnumerator = _columns.GetEnumerator();
+            bool columnMoveNext = columnEnumerator.MoveNext();
+            if (row != null)
+            {
+                IEnumerator<object> rowEnumerator = row.GetEnumerator();
+                bool rowMoveNext = rowEnumerator.MoveNext();
+                while (columnMoveNext && rowMoveNext)
+                {
+                    BaseColumn column = columnEnumerator.Current;
+                    object value = rowEnumerator.Current ?? null;
```

Comment:
```suggestion
                    object value = rowEnumerator.Current;
```
I think the `?? null` is not necessary.